### PR TITLE
Android interop test: Use proto plugin to generate the needed code.

### PR DESCRIPTION
**Pull request by madongfly**
@ejona86 @zhangkun83

### CODE REVIEW

**Comment by zhangkun83** [comment]
I am going to push the protobuf-gradle-plugin to Maven Central today, so that you can depend on it.

File: android-interop-testing/app/build.gradle [hunk 1]
```diff
@@ -19,6 +20,36 @@ android {
...
+    }
+    generateProtoTasks {
+        all().each { task ->
+            task.builtins {
+                javanano {
+                    // Options added to --javanano_out
+                    option 'ignore_services=true'
+                }
+
+                task.plugins {
```
Comment (ejona86) [inline android-interop-testing/app/build.gradle]: The nesting here is a bit strange. It seems that there should be another } before task.plugins.

**Comment by ejona86** [comment]
@madongfly, LGTM if it looks good to @zhangkun83

File: android-interop-testing/app/build.gradle (hunk 1 above)
Comment (zhangkun83) [inline android-interop-testing/app/build.gradle]: Agreed. task.plugins should be in the same level as task.builtins.

Comment (madongfly) [inline android-interop-testing/app/build.gradle]: The task.plugins is nested in task.builtins, this was added by following Kun's suggestion.

@zhangkun83, want to comment?

File: android-interop-testing/build.gradle [hunk 2]
```diff
@@ -3,9 +3,12 @@
 buildscript {
     repositories {
         jcenter()
+        mavenLocal()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:1.1.0'
+        // TODO(madongfly): Change to the released version once it is released.
+        classpath "com.google.protobuf:protobuf-gradle-plugin:0.5.0-SNAPSHOT"
```
Comment (zhangkun83) [inline android-interop-testing/build.gradle]: I have just pushed 0.5.0 to Maven Central. Please update the version and change mavenLocal() to mavenCentral() in the repositories block.

Comment (madongfly) [inline android-interop-testing/build.gradle]: Done.

**Comment by madongfly** [comment]
@zhangkun83 PTAL, thanks!

**Comment by zhangkun83** [comment]
LGTM

**Pull request merged**